packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -21,6 +45,152 @@ export const configSchema = {
       _description:
         'Disable the default "keyup search" for instant patient search as typing concludes on tablet devices',
     },
+    searchFields: {
+      _type: Type.Object,
+      _description: 'Configuration for advanced search fields',
+      _default: {
+        fields: {
```

Comment:
Why do we have `fields` under `searchFields`?
Also we should rename `searchFields` to `searchFilterFields`.
Thanks!

Reply:
Those are the default/built in fields i.e `gender`, `dateOfBirth`, `age`, `postcode`, which can be enabled or disabaled.

Reply:
Yes, but can't we directly have these config points under `searchFields`, rather than `searchFields.fields`